app/javascript/packages/document-capture/components/document-side-acuant-capture.tsx

```diff
@@ -19,6 +19,11 @@ interface DocumentSideAcuantCaptureProps {
   errors: FormStepError<{ front: string; back: string; selfie: string }>[];
   onError: OnErrorCallback;
   className?: string;
+  selfieCaptureEnabled: boolean;
+}
+
+function isUploadAllowed(selfieCaptureEnabled) {
```

Comment:
Can we add TypeScript type annotations, at least for the parameters to avoid them having an `any` type? (return value type could be inferred)

```suggestion
function isUploadAllowed(selfieCaptureEnabled: boolean) {
```

Reply:
Thank you for pointing this out, @aduth! After talking about this PR with Eileen I might remove this method completely or approach this work another way. But I am going to commit this for now anyway in case we don't end up doing either of those things. I appreciate the tips and suggestions to follow Typescript conventions!